mri/operators/_base.py

```diff
@@ -0,0 +1,40 @@
+# #############################################################################
```

Comment:
As discussed, we don't want to have `_base` but `base`.

Reply:
yes on it